strings/revenant_names.json

```diff
@@ -0,0 +1,53 @@
+{
+    "spirit_type": [
+        "Spirit",
+        "Ghost",
+        "Spectre",
+        "Phantom",
+        "Revenant",
+        "Essence",
+        "Poltergeist",
+        "Soul"
+    ],
+
+    "adjective": [
```

Comment:
Let's get "spiteful" as a choice too. Also, can you sort the options by alphabetical order? Would make it easier to read for those of us with shit brains

Reply:
Spiteful added. All lists were alphabetically sorted.